Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -1093,7 +1137,11 @@ void AsyncScrollingCoordinator::windowScreenDidChange(PlatformDisplayID displayI
 
 bool AsyncScrollingCoordinator::hasSubscrollers() const
```

Comment:
It seems that this can only be answered correctly for the main frame.

Reply:
Updated this to include root frame id.